main.go

```diff
@@ -109,3 +117,27 @@ func main() {
 		log.Fatal(err)
 	}
 }
+
+func cleanupPendingBuilds(curfs FileSystem) error {
```

Comment:
How about we call these zombie builds? Maybe `PruneZombieBuilds` is a better name?

Reply:
Sure, I updated the name. :+1: